glslc/README.asciidoc

```diff
@@ -252,6 +252,34 @@ code, such as identifier names and line numbers.
 NOTE: Currently this option has no effect.  Full functionality depends on
 glslang support for generating debug info.
 
+==== `-mfmt=<format>`
+
+Select output format for compilation output in SPIR-V binary code form.
```

Comment:
Missing sentence subjects. Please follow how the other command line options are described.